Code/GraphMol/FileParsers/ProximityBonds.cpp

```diff
@@ -24,12 +24,14 @@ static const double MAXDIST2 = 29.7025;  // MAXDIST*MAXDIST
 
 struct ProximityEntry {
   float x, y, z, r;
-  int atm, hash, next;
+  int atm, hash, next, elem;
 
   bool operator<(const ProximityEntry &p) const { return x < p.x; }
 };
 
-static bool IsBonded(ProximityEntry *p, ProximityEntry *q) {
+static bool IsBonded(ProximityEntry *p, ProximityEntry *q, unsigned int flags) {
+  if (flags & ctdIGNORE_H_H_CONTACTS && p->elem == 1 && q->elem == 1)

```

Comment:
Do we want to throw a warning here perhaps with the PDB serial number and names?

```
WARNING: Ignoring close H-H contact at serial index:989  atom:HG1 and serial indeed:990 atom:HG2 (0.8A)
```